server/chat.ts

```diff
@@ -886,18 +886,19 @@ export class CommandContext extends MessageContext {
 		(this.room || Rooms.global).modlog(entry);
 	}
 	parseSpoiler(str: string) {
-		let privateReason = "";
-		if (!str) return {publicReason: "", privateReason};
+		let publicAndPrivateReason = "";
+		if (!str) return {publicReason: "", publicAndPrivateReason};
 
 		let publicReason = str;
```

Comment:
```suggestion
		if (!str) return {publicReason: "", publicAndPrivateReason: ""};

		let publicReason = str, publicAndPrivateReason = str;
```

Now you can just use `publicAndPrivateReason` instead of `publicAndPrivateReason || publicReason`.

Reply:
Also you can just call this `privateReason`.